agent/src/Ingest.php

```diff
@@ -22,16 +22,16 @@ class Ingest
     private ?TimerInterface $flushBufferAfterDelayTimer = null;
 
     /**
+     * @param  Browser  $browser
      * @param  (Closure(ResponseInterface $response, float $duration): mixed)  $onIngestSuccess
      * @param  (Closure(Throwable $e, float $duration): mixed)  $onIngestError
      */
     public function __construct(
-        private Browser $browser,
+        private $browser,
```

Comment:
What's the reason to remove the type hinting here?

Reply:
The reason we have to do it is because when we build the phar it namespaces everything.

So the typehint that is usually `Laravel\NightwatchAgent\Contracts\Browser` becomes `SomeRandomString123\Laravel\NightwatchAgent\Contracts\Browser`.

In our unit tests we want to provide an implementation of the contract (the fake) but because it is typed and the namespace changes we can't.

So I removed it to allow us to pass a fake instance to the source and the phar.

Reply:
Ah, got it! Thank you for the explanation 👍